### DISCUSSION TITLE
Is there any logs if I use auto-instrument right?

### DISCUSSION THREAD

**Discussion by changliu-wk** [open-telemetry/opentelemetry-python-contrib#1579] [Q&A] [answered]
I am using auto-instrumentation with my services, but I can't see any logs, and I can't find my service from our third party end, I want to know what's wrong, but without any logs, I am in dark. 
I wonder if the agent set up successfully, is there any logs I can see? Or is there a configuration to let me see more logs? 

Thanks!

**Comment by aabmass** [comment, marked as answer]
Can you share the versions of relevant opentelemetry packages from `pip freeze`? Also, what do you mean by logs here, like print statements or `logging` standard library logs?

There was this bug https://github.com/open-telemetry/opentelemetry-python/issues/2594 bug it was fixed quite a while ago. It shouldn't be needed, but you can try adding `--logs_exporter=console`.

**Comment by changliu-wk** [comment]
My problem got solved later, I think that is because I actually didn't install the otel-packages. Use `pip freeze` to check the packages is a right way.
